ReactWindows/ReactNative.Shared/Views/TextInput/ReactTextBox.cs

```diff
@@ -104,6 +133,57 @@ public int IncrementEventCount()
             return Interlocked.Increment(ref _eventCount);
         }
 
+#if WINDOWS_UWP
+        protected override void OnApplyTemplate()
```

Comment:
I doubt that protection level is something that will make the build fail. It is not part of the method signature. So I guess you can just use public and skip the #if here

Reply:
Each one of these #if is needed. Build fails without any one of them.

Reply:
Besides the build, did you test TextInput in WPF Playground with these changes?